QUICKSTART.md

```diff
@@ -0,0 +1,354 @@
+# **KubeStellar** Quickstart
+
+## Required Packages:
+   - [docker](https://docs.docker.com/engine/install/)
+   - [kind](https://kind.sigs.k8s.io/)
+   - [kubectl](https://kubernetes.io/docs/tasks/tools/) (version range expected: 1.23-1.25)
+
+## Setup Instructions
+
+Table of contents:
+
+- [1. Install and run **KubeStellar**](#1-install-and-run-kubestellar)
+- [2. Example deployment of Apache HTTP Server workload into two local kind clusters](#2-example-deployment-of-apache-http-server-workload-into-two-local-kind-clusters)
+  - [a. Stand up two kind clusters: florin and guilder](#a-stand-up-two-kind-clusters-florin-and-guilder)
+  - [b. Create a KubeStellar Inventory Management Workspace (IMW) and Workload Management Workspace (WMW)](#b-create-a-kubestellar-inventory-management-workspace-imw-and-workload-management-workspace-wmw)
+  - [c. Onboarding the clusters](#c-onboarding-the-clusters)
+  - [d. Create and deploy the Apache Server workload into florin and guilder clusters](#e-create-and-deploy-the-apache-server-workload-into-florin-and-guilder-clusters)
+- [3. Teardown the environment](#3-teardown-the-environment)
+
+
+This guide is intended to show how to quickly bring up a **KubeStellar** environment with its dependencies from a binary release.
+
+## 1. Install and run **KubeStellar**
+
+KubeStellar works in the context of kcp, so to use KubeStellar you also need kcp. Download the kcp and **KubeStellar** binaries and scripts into a `kubestellar` subfolder in your current working directory using the following command:
+
+```shell
+kubestellar-bootstrap.sh --kcp-version v0.11.0 --kubestellar-version v0.1.0 --folder .
```

Comment:
The user has not been told to fetch this file, this command can not work.

Reply:
I don't understand this comment. Could you clarify? the `kubestellar-bootstrap.sh` already have a flag for the kubestellar version, isn't it?

Reply:
Earlier drafts of this document exhibited a command that fetched the `kubestellar-bootstrap.sh` from github and fed it into `bash`.  Something like that is needed because there is nothing earlier that downloads anything.

Reply:
ok, got it - just fixed it!